docs/layout-doc.md

```diff
@@ -4,59 +4,62 @@ The Layout layer is where thing get a bit more concrete, it might/should be extr
 
 Every layout in the library has to implement the `ELayout` interface which forces them to implement the following:
 
-`val childLayouts: List<ELayout>` : List of child layout. Can be empty and/or mutable
+`val childLayouts: List<ELayout>` : List of child layouts. Can be empty and/or mutable
+
+//TODO: what does being given meant/size you actually need?
 
 `fun size(toFit: ESize): ESize`
-It's the equivalent of Android's `onMeasure`, your being given and you need to return the size you actually need.
+It's the equivalent of Android's `onMeasure()`, you're being given and you need to return the size you actually need.
```

Comment:
what does "being given"/"size you actually need" mean?

Reply:
It's not clear indeed. What I mean is that an ESize is passed as a parameter, that's the size you have to constrain yourself to, and you have to return the ESize you want to be, according to this constraint